EIPS/eip-6051.md

```diff
@@ -0,0 +1,469 @@
+---
+eip: 6051
+title: Private Key Encapsulation
+description: defines a specification for encapsulating private keys.
+author: Base Labs (@Base-Labs), Weiji Guo (@weiji-cryptonatty)
+discussions-to: https://ethereum-magicians.org/t/private-key-encapsulation-to-move-around-securely-without-entering-seed/11604
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-11-21
+---
+
+
+## Abstract
+
+This EIP proposes a mechanism to encapsulate a private key so that it could be securely relocated to another application without providing the seed. This EIP combines `ECIES` (Elliptic Curve Integrated Encryption Scheme) and optional signature verification under various choices to ensure that the private key is encapsulated to a known or trusted party.
```

Comment:
```suggestion
This EIP proposes a mechanism to encapsulate a private key so that it could be securely relocated to another application without providing the seed. This EIP combines `ECIES` (Elliptic Curve Integrated Encryption Scheme) and optional signature verification under various choices to ensure that the private key is encapsulated by a known or trusted party.
```

Reply:
I was just suggest to use `for` here. It looks like using `by` will give people an impression that the known or trusted party executes the action of the encapsulation.